2022-01-18 20:36:49,066 INFO    MainThread:2304 [wandb_setup.py:_flush():71] setting env: {}
2022-01-18 20:36:49,066 INFO    MainThread:2304 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 20:36:49,066 INFO    MainThread:2304 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 20:36:49,066 INFO    MainThread:2304 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220118_203649-2c5e75g6\logs\debug.log
2022-01-18 20:36:49,067 INFO    MainThread:2304 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220118_203649-2c5e75g6\logs\debug-internal.log
2022-01-18 20:36:49,076 INFO    MainThread:2304 [wandb_init.py:init():404] calling init triggers
2022-01-18 20:36:49,076 INFO    MainThread:2304 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-18 20:36:49,076 INFO    MainThread:2304 [wandb_init.py:init():449] starting backend
2022-01-18 20:36:49,076 INFO    MainThread:2304 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-18 20:36:49,081 INFO    MainThread:2304 [backend.py:ensure_launched():199] starting backend process...
2022-01-18 20:36:49,414 INFO    MainThread:2304 [backend.py:ensure_launched():204] started backend process with pid: 22480
2022-01-18 20:36:49,414 INFO    MainThread:2304 [wandb_init.py:init():458] backend started and connected
2022-01-18 20:36:49,415 INFO    MainThread:2304 [wandb_init.py:init():519] updated telemetry
2022-01-18 20:36:49,422 INFO    MainThread:2304 [wandb_init.py:init():549] communicating current version
2022-01-18 20:36:50,777 INFO    MainThread:2304 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-18 20:36:50,777 INFO    MainThread:2304 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-18 20:36:50,987 INFO    MainThread:2304 [wandb_init.py:init():592] starting run threads in backend
2022-01-18 20:36:51,945 INFO    MainThread:2304 [wandb_run.py:_console_start():1816] atexit reg
2022-01-18 20:36:51,946 INFO    MainThread:2304 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-18 20:36:51,946 INFO    MainThread:2304 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-18 20:36:51,947 INFO    MainThread:2304 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-18 20:36:51,947 INFO    MainThread:2304 [wandb_init.py:init():619] run started, returning control to user process
2022-01-18 20:36:51,948 INFO    MainThread:2304 [wandb_run.py:finish():1463] finishing run austinbergstrom/places_dry_run/2c5e75g6
2022-01-18 20:36:51,948 INFO    MainThread:2304 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-01-18 20:36:51,948 INFO    MainThread:2304 [wandb_run.py:_restore():1758] restore
2022-01-18 20:36:52,451 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-18 20:36:53,079 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-18 20:36:53,299 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-18 20:36:54,007 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4304
}

2022-01-18 20:36:54,111 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4330
}

2022-01-18 20:36:54,212 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4330
}

2022-01-18 20:36:54,314 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4330
}

2022-01-18 20:36:54,415 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3981
  total_bytes: 4330
}

2022-01-18 20:36:54,521 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4330
  total_bytes: 4330
}

2022-01-18 20:36:54,623 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4330
  total_bytes: 4330
}

2022-01-18 20:36:54,728 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4330
  total_bytes: 4330
}

2022-01-18 20:36:54,835 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4330
  total_bytes: 4330
}

2022-01-18 20:36:55,141 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4330
  total_bytes: 4330
}

2022-01-18 20:36:55,702 INFO    MainThread:2304 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4330
  total_bytes: 4330
}
local_info {
}

2022-01-18 20:36:56,823 INFO    MainThread:2304 [wandb_run.py:_append_files():2184] logging synced files
